### PR TITLE
Pull request for adding activemq-camel

### PR DESCRIPTION
**Pull request by henryw30**
there are comments at the bottom for a more concise list of the files that are being added

### CODE REVIEW

**Comment by winglam** [comment]
Why are there new dt list files in your pull request to add activemq? Are you trying to move the dt-list files? If so, why did you not remove the ones from $DT_ROOT/? Also, if you move these files you will also need to update the setup-vars script https://github.com/winglam/dependent-tests-impact/blob/master/experiments/scripts/setup-vars.sh

In the future, changes like this refactoring should be in a separate pull request by itself.

File: experiments/scripts/subj-constants.sh [hunk 1]
```diff
@@ -2,95 +2,103 @@
 rootDir=$1
 # The name of the subject you wish to add (e.g., xml_security).
 expName=(
-#  crystal
```
Comment (winglam) [inline experiments/scripts/subj-constants.sh]: Why are you making changes to this file? In the future, please double check your changes before your pull request. Especially for pull requests that are this big and difficult to fully digest.

**Comment by winglam** [comment]
The initial information you provided is already quite helpful but in the future and after you make your changes, please provide a list of files that you modified, added, and deleted. Please separate the lists.

File: experiments/scripts/subj-constants.sh (hunk 1 above)
Comment (henryw30) [reply experiments/scripts/subj-constants.sh]: activemq and its paths need to be added to this file, but it looks like I commented out the paths for the other projects when I was testing the guidelines. Do you want me to commit the file again with all of the projects uncommented?

**Comment by henryw30** [comment]
The PRIO_DT_LISTS, etc variables in the setupvars script were already configured to point to experiments/scripts/data/prioritization-dt-list (etc), which is why we decided to move the dt-list folders into scripts/data ($DT_DATA)

**Comment by winglam** [comment]
Please resolve your conflicts.

**Review by winglam — APPROVED** [review]
LGTM. Thanks for these changes.

**Pull request merged**